### ISSUE TITLE
Style only Card component

### ISSUE THREAD

**Issue by kaumac** [kumaaa-inc/shadow-panda#189]
It'd be nice to have a Card component that doesn't depend on Context/state.
Right now if I just follow the instructions from the docs I get a warning asking me to add "use client" to my component/page (which is something I don't want).

It'd be nice to have the option to use only the style of the card. I tried this but it doesn't render any style:
```import { styled } from '@/styled/jsx'
import { card } from '@/styled/recipes'

export const Card = styled('div', card)```

**Comment by nanopx** [comment]
Hi @kaumac, thanks for the issue 👍

The problem is that panda currently doesn’t support slot recipes + compound components without using context.
https://panda-css.com/docs/concepts/slot-recipes#styling-jsx-compound-components

I think I should consider using normal config recipes with CSS selectors instead of using slot recipes for these components to avoid the `use client` directive:

- Alert
- Card
- Form
- Table

In the meantime, please consider using the code below for the Card component which doesn’t depend on the context as a workaround(just moved the styles from the slot recipe to styled() 😁):

```tsx
import { styled } from ‘@shadow-panda/styled-system/jsx’

export const Card = styled(‘div’, {
  base: {
    rounded: ‘lg’,
    border: ‘base’,
    bg: ‘card’,
    color: ‘card.foreground’,
    shadow: ‘sm’,
  },
})
export const CardHeader = styled(‘div’, {
  base: {
    display: ‘flex’,
    flexDirection: ‘column’,
    spaceY: ‘1.5’,
    p: ‘6’,
  },
})
export const CardTitle = styled(‘h3’, {
  base: {
    textStyle: ‘2xl’,
    fontWeight: ‘semibold’,
    leading: ‘none’,
    tracking: ‘tight’,
  },
})
export const CardDescription = styled(‘p’, {
  base: {
    textStyle: ‘sm’,
    color: ‘muted.foreground’,
  },
})
export const CardContent = styled(‘div’, {
  base: {
    p: ‘6’,
    pt: ‘0’,
  },
})
export const CardFooter = styled(‘div’, {
  base: {
    display: ‘flex’,
    alignItems: ‘center’,
    p: ‘6’,
    pt: ‘0’,
  },
})
```

**Comment by kaumac** [comment]
Thanks for the quick response @nanopx !
I'm still getting the hang of Panda, could you tell me what's the benefit of using slot recipes + compound components vs. using just normal recipes?

I think using "use client" is fine on more complex components where you would have to use state or context anyways (line a drawer or tabs), but for purely "esthetic" components like a card or a button it'd be preferable not to have to use context/state. (unless there's a big tradeoff on using normal recipes of course)

**Comment by kaumac** [comment]
Btw, I don't know if it's possible, but I think another component that would be a great candidate to be stateless/contextless would be the Navigation Menu.

I think Navigation Menu would be perfect for building landing pages where performance is more critical (for SEO purposes) so being able to render a 100% pure server component would be ideal in this case.

Again, not sure if this is technically possible with the Panda implementation 😄

**Comment by nanopx** [comment]
@kaumac 

> what’s the benefit of using slot recipes + compound components vs. using just normal recipes?

IMO, slot recipes are great when we have a style variant prop(s) on a "root" component, and the children’s styles change by that variant on the root (through context). Also, another benefit for the slot recipe is that we only need 1 recipe to style all the components inside the compound component.

On the other hand, normal recipes are great when styling a single element. Using normal recipes in a compound component will have to use multiple recipes to style each components, and handle the variant from the root component through CSS selectors to apply the styles to the child component.

- Card is a compound component but it doesn’t have any variant prop
- Button has a variant prop, but is not a compound component

So these components shouldn’t have any downsides on using normal recipes, except that the import statement will be a little bit longer since we need a recipe for each element:

```tsx
import { styled } from '@shadow-panda/styled-system/jsx'
import {
  card,
  cardHeader,
  cardTitle,
  cardDescription,
  cardContent,
  cardFooter,
} from '@shadow-panda/styled-system/recipes'

export const Card = styled('div', card)
export const CardHeader = styled('div', cardHeader)
export const CardTitle = styled('h3', cardTitle)
export const CardDescription = styled('p', cardDescription)
export const CardContent = styled('div', cardContent)
export const CardFooter = styled('div', cardFooter)
```


> Btw, I don’t know if it’s possible, but I think another component that would be a great candidate to be stateless/contextless would be the Navigation Menu.

We can’t avoid the `'use client'` directive from the Navigation Menu, since Radix’s navigation menu depends on the context.

**Comment by nanopx** [comment]
Fixed in #191 😄
